tsconfig.json

```diff
@@ -12,8 +12,9 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noImplicitReturns": true,
+    "noImplicitReturns": false,
```

Comment:
I turned this off, because it was wanting a return type for the styles and the render functions in the lit-element components. We can turn it back on, but that seems like syntactic noise to me. Personally. Let me know your thoughts!

Reply:
Good catch!